docs/conf.py

```diff
@@ -92,12 +86,10 @@
 
 pygments_style = "sphinx"
 
-
 # -- Options for HTML output -------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
 html_theme = "sphinx_rtd_theme"
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
```

Comment:
This function was deprecated in the RTD 3.0.0 release: https://github.com/readthedocs/sphinx_rtd_theme/blob/7b894f5e7a5053dd35de406bc779020c229f20a9/sphinx_rtd_theme/__init__.py#L22-L26